docs/DeveloperGuide.md

```diff
@@ -623,6 +623,16 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Import trackAScholarImport File
+
+1. Run trackAScholar program (make some changes to the file) and close it
```

Comment:
Run TrackAScholar instead of run trackAScholar program

Reply:
Fixed it